eng/build.ps1

```diff
@@ -410,6 +410,14 @@ function TestUsingOptimizedRunner() {
   $dlls = $dlls | ?{ -not ($_.FullName -match ".*\\ref\\.*") }
   $dlls = $dlls | ?{ -not ($_.FullName -match ".*/ref/.*") }
 
+  if ($configuration -eq 'Debug') {
+    $excludedConfiguration = 'Release'
+  } else {
+    $excludedConfiguration = 'Debug'
+  }
+
+  $dlls = $dlls | ?{ -not (($_.FullName -match ".*\\$excludedConfiguration\\.*") -or ($_.FullName -match ".*/$excludedConfiguration/.*")) }
```

Comment:
Is there a reason we have to use an `$excludedConfiguration`, instead of including only those dlls which *do* match the `$configuration` we want?

Reply:
This way seemed less prone to accidentally excluding something we meant to include but the output structure is slightly different.